sdks/java/extensions/sql/datacatalog/build.gradle

```diff
@@ -37,6 +36,8 @@ dependencies {
   // Dependencies for the example
   provided project(":sdks:java:io:google-cloud-platform")
   provided library.java.slf4j_api
+
+  testCompile project(":sdks:java:extensions:sql:zetasql")
```

Comment:
nit: The only reference to zetasql is via reflection, so this can be `testRuntimeOnly` right?

Reply:
Ah, you are right. An artifact of an intermediate state of the PR in which I had an enum that returned an actual class.

Reply:
Err, not intermediate state of this PR, but the fact that it was part of #9835 where I use actual classes to eliminate the possibility of a typo.